TsRandomizer/Archipelago/SlotDataParser.cs

```diff
@@ -27,9 +27,9 @@ public static Requirement GetPyramidKeysGate(string pyramidKeysGate)
 			//TODO: remove when clients & server are update with correct value
 			if (pyramidKeysGate == "GateMilitaryGate")
 				return Requirement.GateMilitaryGate;
-			if (pyramidKeysGate == "GateLakeSirineLeft")
+			if (pyramidKeysGate == "GateLakeSereneLeft")
 				return Requirement.GateLakeSereneLeft;
-			if (pyramidKeysGate == "GateLakeSirineRight")
+			if (pyramidKeysGate == "GateLakeSereneRight")
```

Comment:
We cant change these, this is part of a contract between Archipelago Timespinner ServerSide and ClientSide. i specificly added these overwrites to fix it one day when both client and server support the correct names

Reply:
Ahh, ok. Didn't know of that, but makes sense. In that case, I'll go ahead and bin this push request!

Reply:
Ok, thanks for the effort anyway!